loki/README.md

```diff
@@ -0,0 +1,130 @@
+
+    ```bash
+    export KYMA_LOKI_EXAMPLE_NS="{namespace}"
+    ```
+
+2. Export the Helm release name that you want to use. It can be any name, but be aware that all resources in the cluster will be prefixed with that name. Replace the `{release-name}` placeholder in the following command and run it:
+    ```bash
+    export HELM_RELEASE_NAME="{release-name}"
+    ```
+
+3. Update your Helm installation with the required Helm repository:
+
+    ```bash
+    helm repo add grafana https://grafana.github.io/helm-charts
+    helm repo update
+    ```
+
+### Install the loki-stack
+
+Run the Helm upgrade command, which installs the chart if not present yet.
+ ```bash
+helm upgrade --install --create-namespace -n ${KYMA_LOKI_EXAMPLE_NS} ${HELM_RELEASE_NAME} grafana/loki-stack -f https://raw.githubusercontent.com/kyma-project/examples/main/loki/loki-values.yaml --set promtail.enabled=false --set grafana.enabled=false
+```
+
+You can either use the [loki-values.yaml](./loki-values.yaml) provided in this loki folder, which contains customized settings deviating from the default settings, or create your own values.yaml file.
+
+
+### Verify the installation
+
+Check that the `loki` Pod have been created in the `{namespace}` and are in the `Running` state:
+
+```bash
+kubectl -n ${KYMA_LOKI_EXAMPLE_NS} get pod ${HELM_RELEASE_NAME}-0
+```
+
+### Install LogPipeline with custom plugin
+
+> **NOTE:** Before applying following command, `{release-name}` and `{namespace}` placeholder should be replaced.  
+
+Run the kubectl apply command to deploy custom LogPipeline to emmit logs to the deployed loki instance.
+
+Command below will deploy a LogPipeline with `fluentbit` and loki plugin
+
+ ```bash
+kubectl apply -f  https://raw.githubusercontent.com/kyma-project/examples/main/loki/logpipeline-custom.yaml
+```
+
+> **NOTE:** Installed LogPipeline by default will collect all kubernetes labels for collected logs. Please consider to use best practices described in this [guide](https://grafana.com/docs/loki/latest/best-practices/) for best experience with Loki.
+
+### Accessing Loki and Logs
+
+To access Loki, use kubectl port forwarding run:
+```bash
+kubectl -n ${KYMA_LOKI_EXAMPLE_NS} port-forward svc/${HELM_RELEASE_NAME} 3100
+```
+
+Now we can access Loki over `localhost` and query collected logs.
+
+Loki queries need a query parameter `time` in nano second resolution, to get current nano seconds in Linux or MacOS, run following command.
+
+```bash
+date +%s
+```
+
+Replace `{nanoseconds}` placeholder with result of command above and run following command to get latest logs from Loki
+
+```bash
+curl -G -s  "http://localhost:3100/loki/api/v1/query" \
+  --data-urlencode \
+  'query={job="fluentbit"}'
+  --data-urlencode \
+  'time={nanoseconds}'
+```
+
+### Alternative installation options
+
+The used helm chart provides configuration for accessing logs via Grafana, which is disabled by default. To enable Grafana run:
```

Comment:
"used" where?  Is enabling Grafana a prerequisite for a certain procedure? For which procedure?

```suggestion
The used Helm chart provides configuration for accessing logs with Grafana, which is disabled by default. 
To enable Grafana, run:
```